src/config.js

```diff
@@ -0,0 +1,39 @@
+'use strict';
+
+const cfg = require('12factor-config');
+
+const config = cfg({
+  allowedOrigins: {
+    env: 'ALLOWED_ORIGINS',
+    type: 'string',
+    default: '*',
```

Comment:
Please remove the default values. They are contained in the .env.example file; no sense defining them twice. Just replace the `default` property with `required` (with a value to true).

Note: This change will require some other modifications to get the tests to pass, as you'll discover...

Reply:
Defaults in config.js aren't bad, but probably the point should be to set values that are intended for the production app so that you only have a minimal set of env vars specified in production. This is especially true for boilerplate starter code to which you might add new features (and new config vars) in the future. You set sane defaults that don't break the app when the code is updated.